src/lava/magma/compiler/builders/interfaces.py

```diff
@@ -55,11 +55,12 @@ class AbstractProcessBuilder(AbstractBuilder):
     Process builders instantiate and initialize a ProcessModel.
 
     Parameters
-        ----------
-        proc_model: AbstractProcessModel
-                    ProcessModel class of the process to build.
-        model_id: int
-                  model_id represents the ProcessModel ID to build.
+    ----------
```

Comment:
Full stop missing in ln 37, 43, 48 in this file (cannot comment there directly).